bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4727,5 +4727,29 @@
         </sql>
     </changeSet>
 
+    <changeSet id="create-teleconsultation-link-role-202212012000" author="BAHMNI">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM role WHERE role = 'Teleconsultation:CreateLink';
+            </sqlCheck>
+        </preConditions>
+        <comment>Creating new role Teleconsultation:CreateLink for Teleconsultation module</comment>
+        <sql>
+            INSERT INTO role(role, description, uuid) VALUES ("Teleconsultation:CreateLink", "Ability to create teleconsultation link in teleconsultation module", uuid());
```

Comment:
Why is this role needed?

Reply:
@kushboojain does this role is required here ?

Reply:
Removed